workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
See this screenshot:

![workbenchscreenshoticons](https://user-images.githubusercontent.com/9009011/29498420-402b400c-85fc-11e7-9374-4b985e082e53.png)

I would like to have all icons the same size. I do not really specify a size anywhere.

The icons that are small in that screenshot are created with ```g_content_type_get_icon()```. There is no size in that function call. Shortly below that call there is this call: ```gtk_icon_theme_lookup_by_gicon(gtk_icon_theme_get_default(), icon, 16, 0);```. But as I understood this should not change the ```icon```.

The icons that are (too) large in the screenshot have simply been created with ```g_icon_new_for_string()```.

All icons were of small size before I committed the change for supplying different icon size sets. So how do I choose the size? I tried ```g_object_set()``` on the ```GtkTreeView```'s cell renderer (```g_object_set(renderer, "stock-size", 16, NULL);```, the cell renderer is a pixbuf created with ```gtk_cell_renderer_pixbuf_new()```).

Well, enough for today. Maybe I write an mail to the Gtk-App-Devel mailing list tomorrow. Bye for now.